packages/portfolio-deploy/src/chain-info.build.js

```diff
@@ -89,14 +93,23 @@ const parsePeers = strs => {
  *                 .withOpts({rpcAddrs: ['https...]});
  *   const info = await agd.query(['bank', 'balances', 'agoric1...]);
  * @param {{ execFileSync: typeof import('child_process').execFileSync}} io
+ * @param {{ podName?: string, container?: string }} [options] - Optional configuration for kubectl
  */
-const makeAgd = ({ execFileSync }) => {
+const makeAgd = (
+  { execFileSync },
+  { podName = 'agoriclocal-genesis-0', container = 'validator' } = {},
+) => {
```

Comment:
I'm concerned that changing the default behaviour to exec into a pod will not work in general, only for special cases.  Can we instead do something like:

```suggestion
const makeKubeAppd = (io,
  {
    podName = 'agoriclocal-genesis-0',
    container = 'validator',
    appd = 'agd' } = {}
  ) => makeAgd(io, { appPrefix: ['kubectl', 'exec', podName, '-c', container, '--'], appd }); 
const makeAgd = ({ execFileSync }, { appPrefix = [], appd = 'agd' } = {}) => {
  const [appCmd, ...appCmdArgs]= [...appPrefix, appd];
```

Then, clients can use `makeAgd({ execFileSync })` to  get the old behaviour, or `makeKubeAppd({ execFileSync })` to get the new behaviour.

Reply:
@michaelfig, a very valid concern. 

Updated to make it more robust, i.e., looks for `agd` before `kubectl`...